AddressTable

```diff
@@ -0,0 +1,22 @@
+
```

Comment:
Let's change the file name to dbo.Addresses, same for all the rest of these files too.